vtr_flow/scripts/python_libs/vtr/task.py

```diff
@@ -331,10 +347,29 @@ def create_jobs(args, configs, longest_name=0, longest_arch_circuit=0, after_run
             )
 
             if config.sdc_dir:
-                cmd += [
-                    "-sdc_file",
-                    "{}/{}.sdc".format(config.sdc_dir, Path(circuit).stem),
-                ]
+                sdc_name = "{}.sdc".format(Path(circuit).stem)
+                try:
+                    sdc_file = resolve_vtr_source_file(config, sdc_name, config.sdc_dir)
+
+                    cmd += [
+                        "-sdc_file",
+                        "{}".format(sdc_file)
+                    ]
+                except InspectError:
```

Comment:
I don't understand why you are allowing the `except`?

Reply:
In general, It Is possible that some of the tests do not have an sdc file. In such a case, the resolve path function would fail.

Reply:
The original script has `sdc_dir` and `place_constr_dir` to control if the SDC and/or place is expected?  So I don't think this change is right.

Reply:
Ok, I know what is happening. There are two issues here:

1. From symbiflow-arch-defs there are two ways of using the SDC, one is with an explicit constraint file, and the other is through the SDC yosys plugin. In the latter case, the file that gets installed in the benchmarks directory is a symlink, which generates an error when trying to resolve the path.
2. Some tests do not make use of the SDC file (namely counter, murax_full_50, murax_full_100).

Both of these issues should be resolved in arch-defs. We should start cleaning up everything and transition to use the SDC plugin instead of relying on explicit manually written SDCs, as well as adding constraints where they are missing.

For the time being I will temporarily disable tests that do have SDC issues and